### PR TITLE
Setup Circle CI

### PR DESCRIPTION
**Pull request by benjy**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+machine:
+  timezone:
+    Australia/Perth
```
Comment (jibran) [inline circle.yml]: Why not Sydney?

File: composer.json [hunk 2]
```diff
@@ -6,8 +6,8 @@
   "require-dev": {
     "phpunit/phpunit":           "~4.1",
     "phing/phing":               "~2.6",
-    "squizlabs/php_codesniffer": "~1.4",
-    "drupal/coder":              "~7.2"
+    "squizlabs/php_codesniffer": "~2.6",
```
Comment (jibran) [inline composer.json]: With coder 8.2 you don't need this.

File: circle.yml (hunk 1 above)
Comment (benjy) [inline circle.yml]: Why not Perth?

File: composer.json (hunk 2 above)
Comment (benjy) [inline composer.json]: Still listed as a dependency: https://packagist.org/packages/drupal/coder#8.2.7

File: circle.yml (hunk 1 above)
Comment (jibran) [inline circle.yml]: We are using Sydney in all other projects as timezone.

Comment (benjy) [inline circle.yml]: It's good to test on different timezones, it's only relevant to the tests so it doesn't matter unless you have bad code that doesn't work with certain timezones.

**Comment by benjy** [comment]
test this please

**Comment by benjy** [comment]
test this please

File: composer.json (hunk 2 above)
Comment (jibran) [inline composer.json]: Yeah it will be added automatically because of that.

**Pull request merged**